subsys/net/lib/openthread/platform/openthread-core-zephyr-config.h

```diff
@@ -401,4 +401,24 @@
 #define OPENTHREAD_CONFIG_IP6_MAX_EXT_MCAST_ADDRS CONFIG_OPENTHREAD_IP6_MAX_EXT_MCAST_ADDRS
 #endif /* CONFIG_OPENTHREAD_IP6_MAX_EXT_MCAST_ADDRS */
 
+/**
+ * @def OPENTHREAD_CONFIG_TCP_ENABLE
+ *
+ * Enable TCP.
+ *
+ */
+#ifdef CONFIG_OPENTHREAD_TCP_ENABLE
+#define OPENTHREAD_CONFIG_TCP_ENABLE CONFIG_OPENTHREAD_TCP_ENABLE
+#endif /* CONFIG_OPENTHREAD_TCP_ENABLE */
```

Comment:
The default in `openthread` is to [enable it](https://github.com/zephyrproject-rtos/openthread/blob/zephyr/src/core/config/ip6.h#L186).

Maybe always define it?
```C
#define OPENTHREAD_CONFIG_TCP_ENABLE IS_ENABLED(CONFIG_OPENTHREAD_TCP_ENABLE)
```

Reply:
Agree. Same goes for `OPENTHREAD_CONFIG_CLI_TCP_ENABLE`.